README.md

```diff
@@ -12,6 +12,15 @@ Create a new project with a structure like the one seen in the directory `exampl
 It may be stored anywhere on your disk.
 In the root folder of your project, execute `nin run`, and visit http://localhost:8000 in your browser to use nin.
 
+### Compiling
+The `nin compile` command will create a single file `bin/demo.png.html` that contains all the code and resources of your demo. Base64 and PNG compression magic is used to achieve this.
+
+### Start screen
+In the compiled result, you can have a start screen that is shown while the demo is being loaded. When compiling, nin will look for an `index.html` file in the root folder of your demo, and use that as start screen. JavaScript functions `ONPROGRESS`, `ONCOMPLETE` and `STARTDEMO` should be implemented in index.html.
+
+### Render demo to video
+TODO: Needs documentation
```

Comment:
This functionality might not even be in master yet, I think it still lives in its own branch.